tox.ini

```diff
@@ -27,8 +27,9 @@ deps =
     pytest-asyncio
     pytest-xdist
     Twine
+    websockets
     # use fork to pick up fix for https://github.com/aaugustin/websockets/pull/528
-    git+https://github.com/johnsca/websockets@bug/client-redirects#egg=websockets
+    git+https://github.com/johnsca/websockets@bug/client-redirects#egg=websockets ; python_version<'3.9'
 
 [testenv:py{3,35,36,37,38}]
```

Comment:
@johnsca can we also add 39 to this list as well?

Reply:
Ah, this was the reason for the pylxd failure below.  This version-specific section actually isn't necessary; it's redundant with the default testenv, so I just folded them together.